src/Dfc.CourseDirectory.Core/DataStore/Sql/TvpHelper.cs

```diff
@@ -19,5 +19,18 @@ public static SqlMapper.ICustomQueryParameter CreateGuidIdTable(IEnumerable<Guid
 
             return table.AsTableValuedParameter(typeName: "Pttcd.GuidIdTable");
         }
+
+        public static SqlMapper.ICustomQueryParameter CreateIntIdTable(IEnumerable<int> rows)
```

Comment:
Is this ever used? I don't see how it could work since it's referencing `Pttcd.GuidIdTable`

Reply:
No idea how it worked either, though removed for now as i don't think it's required now.